src/Microsoft.Azure.WebJobs.Host/Extensions/AppSettingAttributeExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    internal static class AppSettingAttributeExtensions
+    {
+        public static AutoResolveAttribute ToAutoResolve(this AppSettingAttribute attribute)
+        {
+            return new AutoResolveAttribute
+            {
+                Default = attribute.Default,
+                AllowTokens = false
```

Comment:
That and default values. The rename to `IsAppSetting` with the addition of a `Default` property seems like it would keep things simpler.

@MikeStall any thoughts?